pkg/transform/crio_transform.go

```diff
@@ -174,7 +173,7 @@ func (e CrioTransform) Extract() (Extraction, error) {
 
 	var extraction CrioExtraction
 	if _, err := toml.Decode(string(content), &extraction); err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "Something is wrong in decoding crio")
```

Comment:
Maybe we could use a different wrapping message to be more specific, i.e. "Decoding crio failed"